savant/config/default.yml

```diff
@@ -165,6 +171,14 @@ pipeline:
       source_timeout: ${oc.decode:${oc.env:SOURCE_TIMEOUT, 10}}
       # Interval between source eviction checks (in seconds).
       source_eviction_interval: ${oc.decode:${oc.env:SOURCE_EVICTION_INTERVAL, 1}}
+      # Length of the ingress queue in frames (0 - no limit).
```

Comment:
Please add the default values to the comments.

Reply:
Added.